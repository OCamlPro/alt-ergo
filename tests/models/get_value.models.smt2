(set-option :produce-models true)
(set-option :produce-assignments true)
(set-logic ALL)
(declare-const x Int)
(declare-fun f (Int) Int)
(assert (! (> (f x) 0) :named foo))
(assert (< (f 0) 0))
(check-sat)
(get-value (x (+ x 5)))
(get-assignment)
