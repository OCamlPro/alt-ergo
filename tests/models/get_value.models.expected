
unknown
((x (- 1))
 ((+ x 5) 4))

((foo true))

